src/net/sourceforge/kolmafia/utilities/StringUtilities.java

```diff
@@ -889,6 +889,19 @@ public static final String leetify(final String text) {
     return b.toString();
   }
 
+  public static final boolean isVowel(char letter) {
+    switch (Character.toLowerCase(letter)) {
+      case 'a':
+      case 'e':
+      case 'i':
+      case 'o':
+      case 'u':
+        return true;
```

Comment:
may as well cover this I think, but up to you

Reply:
Sure, will do